src/hooks/useChatsSidebar.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  chatVisibilitySelector,
+  selectedChatSettingsSelector,
+} from "utils/selectors";
+
+import {
+  setPrivateChatTabOpened,
+  setVenueChatTabOpened,
+  setChatSidebarVisibility,
+} from "store/actions/Chat";
+
+import { useSelector } from "./useSelector";
+import { useDispatch } from "./useDispatch";
+import { useNumberOfUnreadChats } from "./usePrivateChats";
+
+export const useChatsSidebarControls = () => {
+  const dispatch = useDispatch();
+
+  const expandChat = () => {
+    dispatch(setChatSidebarVisibility(true));
+  };
+
+  const closeChat = () => {
+    dispatch(setChatSidebarVisibility(false));
+  };
+
+  const openVenueChat = () => {
+    expandChat();
+    dispatch(setVenueChatTabOpened());
+  };
+
+  const openPrivateChats = () => {
+    expandChat();
+    dispatch(setPrivateChatTabOpened());
+  };
+
+  const openPrivateRecipientChat = (recipientId: string) => {
+    expandChat();
+    dispatch(setPrivateChatTabOpened(recipientId));
+  };
+
+  const isChatSidebarVisible = useSelector(chatVisibilitySelector);
+
+  const chatSettings = useSelector(selectedChatSettingsSelector);
+
+  return {
+    isChatSidebarVisible,
+    chatSettings,
+
+    expandChat,
+    openVenueChat,
+    openPrivateChats,
+    openPrivateRecipientChat,
+    closeChat,
+  };
+};
+
+export const useChatsSidebarInfo = () => {
+  const numberOfUnreadChats = useNumberOfUnreadChats();
+
+  return {
+    privateChatTabTitle: `Direct Messages ${
+      numberOfUnreadChats ? `(${numberOfUnreadChats})` : ""
+    }`,
+    venueChatTabTitle: "Venue Chat",
+  };
+};
```

Comment:
It will also have the logic of transforming `venue chat` name into something custom. Are you sure it is better to have it written in `ChatSidebar`?

Reply:
When you say 'will also have', do you mean that that is as yet unwritten logic? If so, perhaps it will make sense in here when it's done. But as the code was written when i reviewed it, this didn't seem like a useful abstraction.

Reply:
Yep, will implement it in this PR. But basically, it will get the custom name of the `Venue chat`, if any

Reply:
Makes sense. And just for my reference. Where does the custom name for the venue chat come from? Is it a venue config thing? Or?

Reply:
I guess. Not sure about it either. I just know that we have such functionality

Reply:
Can you have a look around and let me know? I don't think i've ever actually heard of that being a thing before. So makes me wonder if it's legacy/dead code that we don't need to worry about anymore.

Reply:
Have implemented that